test/graphiteThreshold.check.spec.js

```diff
@@ -33,73 +32,124 @@ describe('Graphite Threshold Check', function(){
 		check.stop();
 	});
 
+	context('Upper threshold enforced', function () {
+
+		it('Should use maxSeries Graphite function to acquire metrics', function (done) {
+			mockGraphite([0]);
```

Comment:
Is what this [commit](https://github.com/Financial-Times/n-health/pull/37/commits/eec54e4610ae2c83f10409ab37b2f5ac5671d521) does what you mean?